frontend/frontend/src/components/Card.vue

```diff
@@ -2,7 +2,9 @@
 	<div class="card">
 		<div v-if="!active"></div>
 		<div v-else-if="status">matched!</div>
-		<div v-else class="card-info">{{ card.cardId }} {{ card.artName }}</div>
+		<div v-else class="card-info">
+			<img style="object-fit: cover" :src="card.artUrl" />
```

Comment:
```suggestion
			<img style="height: 200px; width: 190px" :src="card.artUrl" />
```

Reply:
thanks for suggesting this Abi, I noticed this too!